src/backend/context.rs

```diff
@@ -305,7 +305,7 @@ impl ContextOps for PulseContext {
             user_data: *mut c_void,
         ) {
             let list_data = unsafe { &mut *(user_data as *mut PulseDevListData) };
-            let ctx = &(*list_data.context);
+            let ctx = list_data.context;
```

Comment:
Fix:
```
$  rustup run nightly cargo clippy --all -- -D warnings
error: deref on an immutable reference
   --> src/backend/context.rs:309:11
    |
309 | let ctx = &(*list_data.context);
    |           ^^^^^^^^^^^^^^^^^^^^^ help: if you would like to reborrow, try removing `&*`: `list_data.context`
```

Reply:
it would be great if these clippy fixes are separated into different commits.